modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -37,7 +37,8 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
     if (settings.instSim) {
       if (settings.instForceError) Flamingos2ControllerSimBad
       else Flamingos2ControllerSim
-    } else Flamingos2ControllerEpics
+    } else Flamingos2ControllerEpics,
+    GmosSouthControllerSim
```

Comment:
It should be `GmosSouthControllerSim` or `GmosSouthControllerEpics` depending on `settings.instSim`.